crates/bevy_window/src/window.rs

```diff
@@ -602,12 +602,26 @@ impl Window {
     }
 }
 
+/// Describes the information needed for a window creation like width, height, title, present mode, etc.
```

Comment:
```suggestion
/// Describes the information needed for a window creation.
```
This information is redundant to the info in the field.